cloudformation/athena.yaml

```diff
@@ -102,3 +102,58 @@ Resources:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: aws:kms
               KMSMasterKeyID: !GetAtt KmsKey.Arn
+
+  CalculationDataCrawlerRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              Service:
+                - "glue.amazonaws.com"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
+      Path: "/"
+      Policies:
+        - PolicyName: ResourceTypePolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "logs:AssociateKmsKey"
+                Resource: "*"
+
+  TransactionsCrawlerSecurityConfiguration:
+    Type: AWS::Glue::SecurityConfiguration
+    Properties:
+      EncryptionConfiguration:
+        CloudWatchEncryption:
+          CloudWatchEncryptionMode: SSE-KMS
+          KmsKeyArn: !GetAtt KmsKey.Arn
+        JobBookmarksEncryption:
+          JobBookmarksEncryptionMode: CSE-KMS
+          KmsKeyArn: !GetAtt KmsKey.Arn
+        S3Encryptions:
+          - S3EncryptionMode: SSE-KMS
+            KmsKeyArn: !GetAtt KmsKey.Arn
+      Name: transactions-crawler-security-config
+
+  TransactionsCrawler:
+    Type: AWS::Glue::Crawler
+    Properties:
+      CrawlerSecurityConfiguration: !Ref TransactionsCrawlerSecurityConfiguration
+      DatabaseName: !Ref TransactionDB
+      Description: Crawls the daily buckets of transaction data
+      Name: btm-transactions-crawler
```

Comment:
Is this name going to conflict with other stacks? If in doubt, always use the prefix for names.